tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920120.yaml

```diff
@@ -877,3 +877,75 @@
               -----------------------------265001916915724--
           output:
             no_log_contains: id "920120"
+    -
+      test_title: 920120-36
+      desc: "Attempted multipart/form-data bypass (920120). Still match ; that aren't part of an HTML entity"
+      stages:
+      -
+        stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "ModSecurity CRS 3 Tests"
+              Host: "localhost"
+              Content-Type: multipart/form-data; boundary=---------------------------265001916915724
+            method: POST
+            port: 80
+            uri: /
+            data: |
+              -----------------------------265001916915724
+              Content-Disposition: form-data; name="&amp;fi;zzz"; filename="test"
+              Content-Type: application/octet-stream
+
+              helloworld
+              -----------------------------265001916915724--
+          output:
+            log_contains: id "920120"
+    -
+      test_title: 920120-37
+      desc: "Attempted multipart/form-data bypass (920120). Still match ; that aren't part of an HTML entity."
+      stages:
+      -
+        stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "ModSecurity CRS 3 Tests"
+              Host: "localhost"
+              Content-Type: multipart/form-data; boundary=---------------------------265001916915724
+            method: POST
+            port: 80
+            uri: /
+            data: |
+              -----------------------------265001916915724
+              Content-Disposition: form-data; name="test"; filename="&amp;fi;zzz"
+              Content-Type: application/octet-stream
+
+              helloworld
+              -----------------------------265001916915724--
+          output:
+            log_contains: id "920120"
+    -
+      test_title: 920120-38
+      desc: "Attempted multipart/form-data bypass (920120). Still match ; that aren't part of an HTML entity."
+      stages:
+      -
+        stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "ModSecurity CRS 3 Tests"
+              Host: "localhost"
+              Content-Type: multipart/form-data; boundary=---------------------------265001916915724
```

Comment:
Can you add the `Accept: */*` header to the tests? That way it will not trigger additional rules